helpers/helpers.go

```diff
@@ -518,3 +520,36 @@ func CreateTLSConfig(remoteCAs *x509.CertPool, cert *tls.Certificate) *tls.Confi
 		RootCAs:      remoteCAs,
 	}
 }
+
+// SerializeSCTList serializes a list of SCTs
+func SerializeSCTList(sctList []ct.SignedCertificateTimestamp) ([]byte, error) {
+	var buf bytes.Buffer
+	for _, sct := range sctList {
+		sct, err := ct.SerializeSCT(sct)
+		if err != nil {
+			return nil, err
+		}
+		binary.Write(&buf, binary.BigEndian, uint16(len(sct)))
+		buf.Write(sct)
+	}
+
+	var sctListLengthField = make([]byte, 2)
```

Comment:
To clarify: Do you mean replacing the call to `binary.BigEndian.PutUint16` with `binary.Write`? Wouldn't that force `sctListLengthField` to come after `buf.Bytes()` in the return value?